azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/DeletedSecretItem.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+package com.microsoft.azure.keyvault.models;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The deleted secret item containing metadata about the deleted secret.
+ */
+public class DeletedSecretItem extends SecretItem {
```

Comment:
This is a model from the updated spec we should remove it from the refactoring, and add back in a subsequent update.